src/pudl/transform/ferc1.py

```diff
@@ -5565,6 +5985,15 @@ def table_to_xbrl_factoid_name() -> dict[str, str]:
     }
 
 
+def table_to_column_to_check() -> dict[str, str]:
```

Comment:
The docstring in this helper function is identical to the above helper function.

We already have some helper methods that are defined as properties built into `Ferc1TableTransformParams` that expose values from the bowels of the nested data structure for external use. Would it make sense to add this column in there? Maybe as `params.calculated_column`?

Reply:
i've updated the doc string. I'm going to hold off on this other change just because there is so so much going on in this pr already.